tests/integration/cloud/rest_contracts/conftest.py

```diff
@@ -83,26 +83,6 @@ def cloud_data_context(
     pact_test: pact.Pact,
 ) -> CloudDataContext:
     """This is a real Cloud Data Context that points to the pact mock service instead of the Mercury API."""  # noqa: E501 # FIXME CoP
-    # Set up the /accounts/me contract that CloudDataContext will call during initialization
-    accounts_me_response_body = {
-        "user_id": "df665fc4-1891-4ef7-9a12-a0c46015c92c",
-        "workspaces": [{"id": EXISTING_WORKSPACE_ID, "role": "editor"}],
-    }
-
-    (
-        pact_test.given("the user account exists")
-        .upon_receiving("a request to get user account information")
-        .with_request(
-            method="GET",
-            path=f"/organizations/{EXISTING_ORGANIZATION_ID}/accounts/me",
-            headers={"Authorization": f"Bearer {cloud_access_token}"},
-        )
-        .will_respond_with(
-            status=200,
-            body=accounts_me_response_body,
-        )
-    )
```

Comment:
This is now an unnecessary request. We were implicitly calling `accounts/me` via `CloudDataContext.cloud_user_info()` in the analytics code. Now we only call `accounts/me` if the `workspace_id` has not been provided (to infer the workspace if there is only 1).

Reply:
Thanks, this change would have been confusing without your comment!